example/lib/widgets/buy_sheet.dart

```diff
@@ -280,6 +334,50 @@ class BuySheetState extends State<BuySheet> {
     }
   }
 
+  void _onBuyerVerificationSuccess(BuyerVerificationDetails result) async {
+    if (!_chargeServerHostReplaced) {
+      // In the case of Android, by design, buyer verification happens after card entry completes
+      if (Platform.isIOS) {
+        InAppPayments.completeCardEntry(
+          onCardEntryComplete: _onCardEntryComplete);
+      }
+
+      _showUrlNotSetAndPrintCurlCommand(result.nonce, verificationToken:result.token);
+      return;
+    }
+
+    try {
+      await chargeCardAfterBuyerVerification(result);
+      // In the case of Android, by design, buyer verification happens after card entry completes
+      if (Platform.isIOS) {
```

Comment:
This is for iOS now? what's the behavior on Android?

Reply:
For android, verify() should only be called after the card entry is complete. So at this step, we don't call completeCardEntry again as it's already complete.

Reply:
Synced offline. Android SCA behavior is different from iOS due to limitation in underlying SCA Android library. Updated code to maintain consistency across Android and iOS platforms.